console/src/main/scala/io/shiftleft/console/ScriptManager.scala

```diff
@@ -0,0 +1,32 @@
+package io.shiftleft.console
+
+import java.io.FileInputStream
+import better.files._
+import play.api.libs.json.{Json, Reads}
+
+abstract class ScriptManager(executor: ScriptExecutor) {
+
+  protected val DEFAULT_SCRIPTS_FOLDER: String = "scripts/"
+
+  private val SCRIPT_DESC: String = "description.json"
+
+  case class ScriptDescription(name: String, description: String)
+
+  private implicit val scriptDescriptionRead: Reads[ScriptDescription] = Json.reads[ScriptDescription]
+
+  private def folders() = File(DEFAULT_SCRIPTS_FOLDER).children.filter(_.isDirectory)
+
+  def scripts(): List[ScriptDescription] = {
+    val allScriptDescriptions = folders().map { folder =>
+      Json.parse(new FileInputStream(folder.pathAsString + "/" + SCRIPT_DESC)).as[ScriptDescription]
+    }
+    allScriptDescriptions.toList
+  }
+
+  def runScript(name: String): Object = {
```

Comment:
It might be nice to provide a version of this that allows the user to specify a type parameter, then we can cast the `Object` into any `T` for them.